### DISCUSSION TITLE
Make it possible to call yamlfixer from ymlfxr (alias like approach)

### DISCUSSION THREAD

**Discussion by adriens** [opt-nc/yamlfixer#35] [General]
It would be cool is, when the pypi package is installed, to be able to call it from the following aliases (examples)  : 

- `ymlfxr`
- `ymlfix`

**Comment by adriens** [comment]
What do you thing about that idea @tamere-allo-peter @mbarre ❔

**Comment by tamere-allo-peter** [comment]
```shell
cat ~/.bashrc
...
alias ymlfix='yamlfixer'
alias ymlfxr='yamlfixer'
...
```

I don't see any need for change in yamlfixer.

**Reply by adriens** [reply]
I was almost sure you would push this approach 👍

**Reply by tamere-allo-peter** [reply]
@adriens Never underestimate my lazyness !

**Comment by adriens** [comment]
Discussion unpinned.

**Comment by adriens** [comment]
cf https://github.com/ohmyzsh/ohmyzsh/issues/10788
